poe-api/src/poe.py

```diff
@@ -221,9 +219,14 @@ def get_next_data(self, overwrite_vars=False):
     json_regex = r'<script id="__NEXT_DATA__" type="application\/json">(.+?)</script>'
     json_text = re.search(json_regex, r.text).group(1)
     next_data = json.loads(json_text)
+    key_regex = r'(https://[^<]*_app.*?\.js)'
+    key_url = re.search(key_regex, r.text).group(1)
+    s = request_with_retries(self.session.get, key_url)
```

Comment:
I'd prefer for this to be in its own function rather than lumped into `get_next_data`.